cookiecutter.json

```diff
@@ -27,6 +27,7 @@
   "cloud_provider": [
     "AWS",
     "GCP",
+    "AZURE",
```

Comment:
The other 2 (AWS and GCP) are in capital letters because they are acronyms. I think the name "Azure" is a single word. Shall we change that to `Azure` (not all caps)?

Reply:
Yep, I've updated all the references.